common/src/WooCommerce.php

```diff
@@ -143,6 +133,29 @@ public function order_completed($order_id) {
         }
     }
 
+	public function gtin_wwk_option() {
+		$gtin_handler  = new GtinHandler();
+		if (!empty( $gtin_handler->getActivePlugin() ) ) {
```

Comment:
No need for `!empty` here.  You can just do: `if ($gtin_handler->getActivePlugin()) {`